host_core/test/e2e/kvcounter_test.exs

```diff
@@ -88,4 +92,77 @@ defmodule HostCore.E2E.KVCounterTest do
     {:ok, resp} = HTTPoison.get("http://localhost:8081/foobar")
     assert resp.body == "{\"counter\":#{incr_count}}"
   end
+
+  test "kvcounter unprivileged access denied" do
+    {:ok, bytes} = File.read(@kvcounter_unpriv_path)
+    Logger.info("attempting to load kvcounter actor")
+    Logger.info("Here's the lattice prefix btw: #{HostCore.Host.lattice_prefix()}")
+    {:ok, _pid} = HostCore.Actors.ActorSupervisor.start_actor(bytes)
+    Logger.info("successfully started kvcounter actor")
```

Comment:
These 3 logger.infos were probably just copied from the other test (I wrote it, no shame) but we can remove them since it was just me trying to track down the intermittent test failures